app/qml/main.qml

```diff
@@ -715,6 +715,21 @@ ApplicationWindow {
     onManageAccountClicked: Qt.openUrlExternally(__inputHelp.merginSubscriptionLink)
   }
 
+  MMMonthlyContributorsLimitDialog {
+    id: monthlyContributorsLimitDialog
+
+    apiSupportsSubscription: __merginApi.apiSupportsSubscriptions
+
+    onManageAccountClicked: function() {
+      if (__merginApi.apiSupportsSubscriptions) {
+        Qt.openUrlExternally(__inputHelp.merginSubscriptionLink)
+      }
+      else {
+        Qt.openUrlExternally(__inputHelp.merginPricingLink)
+      }
+    }
```

Comment:
This should be enough, if the server does not support subscription, we do not need to send them to our pricing page.

```suggestion
    onManageAccountClicked: Qt.openUrlExternally( __inputHelp.merginSubscriptionLink )
```